src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -152,19 +150,23 @@ public void test() throws IOException { //NOPMD - assert is done in subs.
         playerDies();
         clickPopup(DEATH_TOMAIN);
 
-//        mainMenu(MAIN_QUIT);
+        //        mainMenu(MAIN_QUIT);
         Platform.runLater(stage::close);
         outputEventLog();
         new File("sav/Fred.ses").delete();
     }
 
-    private void outputEventLog() throws IOException {
-        ShaftEscape.LOGGER.writeToFile();
-        final String log = new String(Files.readAllBytes(
-                Paths.get(Logger.OUTFILE)), StandardCharsets.UTF_8);
-        System.out.println("\n== EVENT_LOG ==");     //NOPMD - Intended use of
-        System.out.println(log);                     //NOPMD - System.out.print
-        System.out.println("== END_EVENT_LOG ==\n"); //NOPMD - for Travis log
+    private void outputEventLog() {
+        try {
+            ShaftEscape.LOGGER.writeToFile();
+            final String log = new String(Files.readAllBytes(
+                    Paths.get(Logger.OUTFILE)), StandardCharsets.UTF_8);
+            System.out.println("\n== EVENT_LOG ==");     //NOPMD - Intended use of
+            System.out.println(log);                     //NOPMD - System.out.print
+            System.out.println("== END_EVENT_LOG ==\n"); //NOPMD - for Travis log
+        } catch (IOException e) {
+            fail();
```

Comment:
Should the E2E really fail when the log cannot be printed to the console? Maybe it's better just to print "log could not be printed" or something like that, including the Exception message.

<review, testing, logic